package.json

```diff
@@ -5,32 +5,32 @@
   "author": "Joyent (joyent.com)",
   "license": "MPL-2.0",
   "private": true,
+  "// dependencies TODO": "artedi to v2 https://github.com/joyent/node-artedi/blob/master/docs/migrating.md",
   "dependencies": {
-    "artedi": "1.3.0",
-    "assert-plus": "0.1.5",
-    "bunyan": "1.5.1",
+    "artedi": "1.4.1",
+    "assert-plus": "1.0.0",
+    "bunyan": "1.8.12",
     "clone": "0.1.1",
-    "dtrace-provider": "0.8.7",
+    "dtrace-provider": "0.8.8",
     "lru-cache": "4.1.5",
     "mname": "1.5.0",
     "mooremachine": "^2.1.0",
     "mname-client": "0.6.0",
     "posix-getopt": "1.0.0",
     "restify": "4.3.2",
-    "sdc-clients": "9.0.3",
     "triton-metrics": "0.1.1",
+    "ufds": "1.6.0",
     "vasync": "2.1.0",
     "verror": "^1.10.0",
     "xtend": "1.0.3",
     "zkstream": "0.11.8"
   },
-  "engines": {
-    "node": ">=0.10"
-  },
   "devDependencies": {
     "expiring-lru-cache": "2.1.0",
```

Comment:
It looks like nothing calls test/helper.js's `createCache` which is the only user of this dependency. Perhaps `createCache` and this dep could be dropped.


Please also consider cleaning up helper.js. It's `before`, `after`, and `test` are vestigial now that nodeunit has been dropped.

Reply:
Ah, ok. I will see if I can drop it, and will do on the cleanup.

Reply:
Done.